src/shared-components/Card/ListItemsSelect/ListItemSelect.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText,
+  Checkbox,
+} from '@mui/material';
+import RadioButtonUncheckedIcon from '@mui/icons-material/RadioButtonUnchecked';
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+
+export const ListItemSelect = ({ title }: { title: string }) => {
```

Comment:
This type of list item would also know about if the list item is selected or not, add a second prop (isSelected) to it and show the checkbox based on the value of the prop.

Reply:
Done